katcp/test/test_resource_client.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
The translator tool does use an `old_div()` workaround if it doesn't know, but in the case where it did it I knew that at least one of the numbers was guaranteed to be a float, so I replaced with with a normal `/`.  But probably not a bad idea to do the imports as you suggest.